src/infra/main/template.yaml

```diff
@@ -1346,6 +1399,146 @@ Resources:
       AliasName: !Sub "alias/${AWS::StackName}-CloudWatchEncryptionKey"
       TargetKeyId: !GetAtt CloudWatchEncryptionKey.Arn
 
+  #
+  #  Canary Deployment CloudWatch Alarms
+  #
+
+  StatusRetrieverFunctionErrorsAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Properties:
+      ActionsEnabled: true
+      AlarmActions:
+        - !Ref AlertingSNSHighAlertNotificationTopicARN
```

Comment:
Not sure this should be a high alert as it takes one occurrence to disturb Live Services where as actually ApiHttp5xxCriticalErrors is for this.